style.css

```diff
@@ -389,12 +392,18 @@ hr {
   border: 2px solid #fff;
 }
 
+.desk-form .btn:active {
+  background-color: #2230d2;
+  color: white;
+  border: 2px solid #fff;
+}
+
 .contact-align hr {
   margin: 0 auto;
   margin-bottom: 8px;
   width: 125px;
   height: 5px;
-  background-color: black;
+  background-color: #fff;
   margin-top: 36px;
   border-radius: 10px;
   border: none;
```

Comment:
- [ ] Good work on your contact form but there are some aspects of the design you can improve on. 
  - Reduce the spacing between the `input elements ` to match the original design

![image](https://user-images.githubusercontent.com/98453979/235647257-dade143c-fcac-46c8-a10b-e3eafa94a2e8.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/235647373-d0e0c3dc-57c2-4fad-b27e-9138a6a9ab93.png)

Reply:
@frank1738 Thanks for the review. I tried to follow the hard measurement given in Figma which was 20px. While I gave 15px and was a bit sceptic about this if not following the exact measurement might become an issue.

I agree that the gaps was not pleasant to look at. I have applied the measurement "11px" now it is look alike. And I am thankful for this advice now it is looking more nearer to the actual design given in figma.

😄

Reply:
> * [ ]  Good work on your contact form but there are some aspects of the design you can improve on.
>   
>   * Reduce the spacing between the `input elements ` to match the original design
> 
> ![image](https://user-images.githubusercontent.com/98453979/235647257-dade143c-fcac-46c8-a10b-e3eafa94a2e8.png)
> 
> ### Checkout the original design
> ![image](https://user-images.githubusercontent.com/98453979/235647373-d0e0c3dc-57c2-4fad-b27e-9138a6a9ab93.png)


- [ ] Kindly note that the previous reviewer was talking about height, not color. To simplify things, I have helped checked further, I noticed there is a <br> in between the inputs, when I removed the <br> tags, the spaces reduced to what the previous reviewer noticed so I would just recommend you remove the <br> tags and we are good to go.

### When I removed the `<br>`
![image](https://user-images.githubusercontent.com/86200356/235678604-82121c3e-464a-4e5b-a122-70b1211d6d3f.png)


### When the `<br>` was there
![image](https://user-images.githubusercontent.com/86200356/235678794-8d186d05-2e73-47b8-8f7b-d1c7f14945b2.png)

Reply:
Fixed.